frontend/sw360-portlet/pom.xml

```diff
@@ -111,6 +112,17 @@
             <version>${project.version}</version>
         </dependency>
 
+    <dependency>
```

Comment:
The webjars-locator provides useful methods to generate necessary requirejs config code from other webjars.